app/views/cart/show.html.erb

```diff
@@ -31,6 +31,14 @@
       <td>
         <center>
           <p><%=quantity%></p>
+          <%= form_tag "/cart/#{item.id}/add_quantity", method: :patch do %>
+          <p><%= number_field_tag 'quantity', '1', min: 1, max: "#{item.inventory - 1}" %></p>
+          <%= submit_tag 'Update Quantity' %>
+          <% end %>
+          <%= form_tag "/cart/#{item.id}/decrease-quantity", method: :patch do %>
+          <p><%= number_field_tag 'quantity_decrease', '1', min: 1, max: "#{item.inventory - 1}" %></p>
+          <%= submit_tag 'Decrease Quantity' %>
+          <% end %>
```

Comment:
I think I could find a way to create a drop-down menu that would all the user to add or subtract quantity in the same form. For now, I have two separate forms.